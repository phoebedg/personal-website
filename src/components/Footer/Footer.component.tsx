import React from "react";

export const Footer: React.FC = () => (
  <div className="footer">
    <span className="footer-text">phoebedg 2020</span>
  </div>
);
