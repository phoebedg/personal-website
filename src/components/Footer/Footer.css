.footer {
    text-align: right;
    padding: 10px;
    background: var(--teal-color);
    margin-top: 60px;
  }
  
  .footer-text {
    padding: 10px;
    display: block;
    color: var(--neutral-color);
    font-weight: 800;
  }
  