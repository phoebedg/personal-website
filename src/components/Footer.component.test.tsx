import React from "react";
import { render } from "@testing-library/react";
import { Footer } from "./Footer.component";

test("renders ListLink", async () => {
  const { container, getByText } = render(<Footer />);
  expect(getByText("phoebedg 2020")).toBeInTheDocument();
  expect(container).toMatchSnapshot();
});
