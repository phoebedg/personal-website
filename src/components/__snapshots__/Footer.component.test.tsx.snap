// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ListLink 1`] = `
<div>
  <div
    class="app__footer"
  >
    <span
      class="app__footer-text"
    >
      phoebedg 2020
    </span>
  </div>
</div>
`;
